crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -253,6 +253,15 @@ impl<T: TransactionOrdering> TxPool<T> {
         // Process the sub-pool updates
         let UpdateOutcome { promoted, discarded } = self.process_updates(updates);
 
+        // Compute pool metrics
+        let stats = self.size();
+        self.metrics.pending_pool_transactions.set(stats.pending as f64);
+        self.metrics.pending_pool_size_bytes.set(stats.pending_size as f64);
+        self.metrics.basefee_pool_transactions.set(stats.basefee as f64);
+        self.metrics.basefee_pool_size_bytes.set(stats.basefee_size as f64);
+        self.metrics.queued_pool_transactions.set(stats.queued as f64);
+        self.metrics.queued_pool_size_bytes.set(stats.queued_size as f64);
+
```

Comment:
this is a pedantic nit, but I'd like to do this in a separate function, like `update_size_metrics` or smth.

Reply:
done :)